### PR TITLE
First Two Sections of the Personal Portfolio Project

### PR DESCRIPTION
**Pull request by DarkxDev**
Created a new project for our Personal Portfolio, including:

- Set up linters and gitignore files.
- Created basic HTML & CSS structure and styling for the two first sections of the webpage.
- Added a folder and image files for the background and icons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [👥 Authors](#authors)
+
```
Comment (Owusu-Desmond) [inline README.md:9]: - [ ] Please make sure that you update your 📗 Table of Contents section as you add the required section I have stated :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @DarkxDev , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
+@media (min-width: 400px) {
```
Comment (GedeonTS) [inline styles.css:1]: - [ OPTIONAL] Kindly remove this media query to allow your application be fit to 375px as per the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), as it has not been pointed out by the previous reviewer it's mentioned as optional, However, I highly suggest this change to you :+1:

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    padding: 15px;
+    align-items: center;
+  }
+
+  #about {
+    background-image: url(imgs/header_shapes_mobile.png);
+  }
+
+  #about h1 {
+    width: 327px;
```
Comment (GedeonTS) [inline styles.css:18]: - [OPTIONAL] In my opinio, I think it would be better if you used `vw`, `vh` and percentages to make your website responsive.

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :+1: 

Hi @DarkxDev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  :clap:  :clap: :clap: 

## Highlights
- Linters are passing :+1: 
- Good PR description :+1: 
- Meaningful commit messages :+1: 

## Optional suggestions

__Kindly check the comments under the review__

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**